khelo/lib/ui/flow/settings/support/contact_support_screen.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+import 'package:khelo/components/app_page.dart';
+import 'package:khelo/components/error_snackbar.dart';
+import 'package:khelo/domain/extensions/context_extensions.dart';
+import 'package:khelo/ui/flow/settings/support/contact_support_view_model.dart';
+import 'package:style/animations/on_tap_scale.dart';
+import 'package:style/button/primary_button.dart';
+import 'package:style/extensions/context_extensions.dart';
+import 'package:style/text/app_text_field.dart';
+import 'package:style/text/app_text_style.dart';
+
+import 'components/attachment_item.dart';
+
+class ContactSupportScreen extends ConsumerStatefulWidget {
+  const ContactSupportScreen({super.key});
+
+  @override
+  ConsumerState<ContactSupportScreen> createState() =>
+      _ContactSupportScreenState();
+}
+
+class _ContactSupportScreenState extends ConsumerState<ContactSupportScreen> {
+  late ContactSupportViewStateNotifier notifier;
+
+  @override
+  void initState() {
+    super.initState();
+    notifier = ref.read(contactSupportStateNotifierProvider.notifier);
+  }
+
+  void _errorObserve() {
+    ref.listen(
+      contactSupportStateNotifierProvider.select(
+        (value) => value.actionError,
+      ),
+      (previous, next) {
+        if (next != null) {
+          showErrorSnackBar(context: context, error: next);
+        }
+      },
+    );
+  }
+
+  void _popObserve() {
+    ref.listen(
+      contactSupportStateNotifierProvider.select(
+        (value) => value.pop,
+      ),
+      (previous, next) {
+        if (next) {
+          context.pop();
+        }
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    _errorObserve();
+    _popObserve();
+    return AppPage(
+      title: context.l10n.contact_support_title,
+      body: Builder(builder: (context) {
+        return _body(context);
+      }),
+    );
+  }
+
+  Widget _body(BuildContext context) {
+    final state = ref.watch(contactSupportStateNotifierProvider);
+    return ListView(
+      padding: context.mediaQueryPadding + const EdgeInsets.all(16),
+      children: [
+        _textInputField(
+          context,
+          placeholderText: context.l10n.contact_support_title_placeholder_text,
+          controller: state.titleController,
+        ),
+        const SizedBox(height: 16),
+        _textInputField(
+          context,
+          placeholderText:
+              context.l10n.contact_support_description_placeholder_text,
+          controller: state.descriptionController,
+          maxLines: 8,
+        ),
+        const SizedBox(height: 16),
+        _attachmentButton(
+          context: context,
+          onAttachmentTap: () {
+            notifier.pickAttachments();
+          },
```

Comment:
use shorter version for optimization:  **onAttachmentTap: notifier.pickAttachments**